modiscolite/util.py

```diff
@@ -2,9 +2,27 @@
 # Authors: Jacob Schreiber <jmschreiber91@gmail.com>
 # adapted from code written by Avanti Shrikumar 
 
+from enum import Enum
 import numpy as np
 from numba import njit
 
+
+class MemeDataType(Enum):
+	PFM = "pfm"
+	CWM = "cwm"
+	hCWM = "hcwm"
+	CWM_PFM = "cwm-pfm"
+	hCWM_PFM = "hcwm-pfm"
+
+	def __str__(self):
+		return self.value
+
+
+def case_insensitive_meme_datatype(arg_str):
```

Comment:
Let's remove this. I think users should just pass in exactly matching strings.

Reply:
Removed